packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -337,5 +324,5 @@ void main() {
     );
 
     expect(textSpanTree, equals(expectedTextSpanTree));
-  });
+  }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.android, TargetPlatform.iOS }));
 }
```

Comment:
What happened to the test that you added in 7904ab2cc4d112071c2a65d16a54d845399a058f?